templates/citizen_project/_email_summary.html.twig

```diff
@@ -0,0 +1,11 @@
+<div>
+    {% for citizen_project in citizen_projects %}
+   <div>
+       <a href="{{ path('app_citizen_project_show', { slug: citizen_project.slug }) }}" title="{{ citizen_project.name }}">
```

Comment:
Since this template is rendered by a worker, you need to generate an absolute url.
Otherwise scheme and/or domain will be missing in the link depending on the configuration.

Please use `url()` instead of `path()`.